### PR TITLE
Move to riff-raff.yaml. 

### PR DESCRIPTION
**Pull request by jennysivapalan**
Giving viewer more ❤️ . 
I've moved us from deploy.json to riff-raff.yaml.
* I've removed updating cloudformation in riff-raff. The advice in the riff-raff documentation is to not do this if an app is continuously deployed (which this is).
* I've added update AMI to the deployment.

### CODE REVIEW

File: riff-raff.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    parameters:
+      bucket: composer-dist
+    dependencies:
+      - viewer-ami-update
+  viewer-ami-update:
+      type: ami-cloudformation-parameter
+      parameters:
+        amiTags:
+          Recipe: editorial-tools-basic
+          AmigoStage: PROD
```
Comment (TBonnin) [inline riff-raff.yaml:15]: can you add the `BuiltBy: amigo` tag. 
I'd recommend to always use it

**Review by TBonnin — APPROVED** [review]
I don't know this stack but this looks sane to me

File: riff-raff.yaml (hunk 1 above)
Comment (jennysivapalan) [reply riff-raff.yaml:15]: What does it do?

Comment (TBonnin) [reply riff-raff.yaml:15]: That's one of the tag added to an AMI by Amigo. I would work without it but it is an additional way to make sure riff-raff discovers the right AMI

Comment (jennysivapalan) [reply riff-raff.yaml:15]: Ok, thanks!

**Pull request merged**